src/docfx/config/AppData.cs

```diff
@@ -35,10 +35,11 @@ public static string GetCommitCachePath(string remote)
             return Path.Combine(CacheRoot, "commits", HashUtility.GetMd5Hash(remote));
         }
 
-        public static string GetCommitBuildTimePath(string remote, string branch)
+        public static string GetCommitBuildTimePath(string remote, string? branch)
         {
+            var branchHash = string.IsNullOrEmpty(branch) ? string.Empty : HashUtility.GetMd5Guid(branch).ToString();
```

Comment:
consider suppress the commit build time feature entirely when there is no branch

Reply:
which feature do you mean? can you add the task here if there is?

Reply:
ContributionProvider.cs line 112 is currently controlling when to enable commit build time.